examples/webgl_materials_compile.html

```diff
@@ -33,205 +33,211 @@
 			Loading ...
 		</div>
 
-		<script src="../build/three.js"></script>
-
-		<script src='js/geometries/TeapotBufferGeometry.js'></script>
-		<script src="js/controls/OrbitControls.js"></script>
-		<script src="js/libs/dat.gui.min.js"></script>
-
 		<script type="module">
 
-		import './js/nodes/THREE.Nodes.js';
+			import * as THREE from '../build/three.module.js';
 
-		var container = document.getElementById( 'container' );
+			import { OrbitControls } from './jsm/controls/OrbitControls.js';
+			import { TeapotBufferGeometry } from './jsm/geometries/TeapotBufferGeometry.js';
+			import { NodeFrame } from './jsm/nodes/core/NodeFrame.js';
+			import { MathNode } from './jsm/nodes/math/MathNode.js';
+			import { OperatorNode } from './jsm/nodes/math/OperatorNode.js';
+			import { ConstNode } from './jsm/nodes/core/ConstNode.js';
+			import { ColorNode } from './jsm/nodes/inputs/ColorNode.js';
+			import { ExpressionNode } from './jsm/nodes/core/ExpressionNode.js';
+			import { TimerNode } from './jsm/nodes/utils/TimerNode.js';
+			import { FloatNode } from './jsm/nodes/inputs/FloatNode.js';
+			import { PhongNodeMaterial } from './jsm/nodes/materials/PhongNodeMaterial.js';
```

Comment:
We don't import individual files from `src/`, but treat it as a single module. I think the same should apply to nodes, and we'd encourage users to write:

```js
import {
  NodeFrame,
  MathNode,
  OperatorNode,
  ...
} from './jsm/nodes/Nodes.js';

```

Reply:
Perhaps `Nodes.js` should be renamed `index.js`?

Reply:
Okay, I change the examples to this style:
```js
import * as Nodes from './jsm/nodes/Nodes.js';
```

Reply:
> We don't import individual files from src/, but treat it as a single module.

Okay, done!

I'm not feeling strong about the name of `Nodes.js`. `index.js` seems to be fine, though.

Reply:
> Perhaps Nodes.js should be renamed index.js?

I vote to keep `Nodes.js` name, maintain the same standard of those on core like `Materials.js` files, that was inspired.
https://github.com/mrdoob/three.js/blob/dev/src/materials/Materials.js
https://github.com/mrdoob/three.js/blob/dev/src/Three.js

`src/nodes/Nodes.js`
`src/materials/Materials.js`
...